lambdas/process_webhooks/constants.py

```diff
@@ -0,0 +1,40 @@
+# Create constants that hold lists of jobs.
```

Comment:
Yes. _get_jobs_list will return [] so it will not query for them. The ticket for this work mentioned only platform jobs. I can add in e2e in the future.